app/src/main/java/org/p2p/wallet/App.kt

```diff
@@ -44,6 +42,8 @@ class App : Application() {
         SolanjLogger.setLoggerImplementation(SolanajTimberLogger())
 
         appCreatedAction.invoke()
+
+        AppsFlyManager.install(this)
```

Comment:
AppsFlyManager  -> AppsflyerManager but it's better to stick with the current naming that we have - AppsflyerService